tests/qiskit_convert_test.py

```diff
@@ -413,7 +440,7 @@ def test_condition_errors() -> None:
         c = Circuit(2, 2)
         c.X(0, condition_bits=[1, 0], condition_value=1)
         tk_to_qiskit(c)
-    assert "OpenQASM conditions must be an entire register in order" in str(
+    assert "OpenQASM conditions must be an entire register in\n order" in str(
```

Comment:
Don't much like having the newline in the error message.

Reply:
done in @cqc-melf
[remove linebreak from error message](https://github.com/CQCL/pytket-qiskit/pull/245/commits/f2a4770491ea359b7c93e021be5d46220611e92a)
[f2a4770](https://github.com/CQCL/pytket-qiskit/pull/245/commits/f2a4770491ea359b7c93e021be5d46220611e92a)